### PR TITLE
[FIX] website_form_recaptcha: Re-recaptcha

### PR DESCRIPTION
**Pull request by chienandalu**
- When website_crm_phone_validation is installed, the captcha validation
is called twice so the form allways fail as with a
'timeout-or-duplicate' error.

cc @Tecnativa

### CODE REVIEW

File: website_form_recaptcha/controllers/main.py [hunk 1]
```diff
@@ -32,6 +32,11 @@ def extract_data(self, model, values):
         res = super(WebsiteForm, self).extract_data(model, values)
         if model.website_form_recaptcha:
             captcha_obj = request.env['website.form.recaptcha']
+            # Only check once
+            try:
+                getattr(request, captcha_obj.RESPONSE_ATTR)
+            except AttributeError:
+                return res
```
Comment (pedrobaeza) [inline website_form_recaptcha/controllers/main.py:43]: Why do you return earlier here?

Comment (chienandalu) [reply website_form_recaptcha/controllers/main.py:43]: The problem here is that with the reCaptcha API, if you make the same call twice, you get an error. So after the first invoke, I store the response's unique value in the request object. So the successive times this is call we'll return as it's already validated.

Comment (pedrobaeza) [reply website_form_recaptcha/controllers/main.py:43]: Why don't you put this as comment in the code then for avoiding wondering this on the future?

**Review by pedrobaeza — APPROVED** [review]

**Review by yajo — APPROVED** [review]
It looks OK. Just one suggestion: Maybe we should just move the check to the `website_form` method?

**Comment by chienandalu** [comment]
I guess it would be valid, although isn't it better to channel the output with rest of the validations?: https://github.com/odoo/odoo/blob/11.0/addons/website_form/controllers/main.py#L33

**Comment by chienandalu** [comment]
The LINT is failing on another module's readme links: https://travis-ci.org/OCA/website/jobs/459663549#L852

**Comment by yajo** [comment]
I think it's this: https://travis-ci.org/OCA/website/jobs/459663549#L1055

OK about https://github.com/OCA/website/pull/540#issuecomment-442103180.

**Comment by chienandalu** [comment]
I'd say is this one: https://travis-ci.org/OCA/website/jobs/459663549#L852

**Comment by yajo** [comment]
Rebase please.

**Comment by chienandalu** [comment]
There we are :+1:

**Pull request merged**